### ISSUE TITLE
Возможность рассрочки без отображения на сайте

### ISSUE THREAD

**Issue by lavkalavkalavka** [yoomoney/cms-opencart15#46]
Клиент выбирает оплату Юкассой, а способы оплаты выбираются на стороне самой Юкассы, а не у нас на сайте. В таком случае кнопка "Заплатить по частям" не имеет смысла.
Каким образом возможно предоставить клиентам оформление рассрочки без отображения блока "Заплатить по частям" в товаре или на этапе оформления?

**Comment by ostulov** [comment]
Добрый день!

Возможно я не совсем понимаю Ваш вопрос, но через модуль возможность оформить рассрочку предоставляется либо в момент выбора способа оплаты на стороне сайта, либо при выборе из списка способов на стороне ЮKassa. На странице товара размещается блок с информацией о рассрочке. Иные способы в модуле не предусмотрены.

**Comment by lavkalavkalavka** [comment]
У меня отсутствовал вариант "Оплата по частям" при переходе на страницу оплаты Юкассы до тех пор, пока я не включил отображение указанного блока на сайте при оформлении заказа. Может ли быть связана возможность оплаты "по частям" с этим блоком?

**Comment by ostulov** [comment]
Нет, на стороне ЮKassa отображаются все доступные магазину способы оплаты.
Показ способа оплаты "Оплата по частям" может зависеть только от суммы всего платежа. Так как данный способ имеет минимальное значение в 3 тысячи рублей, то для платежей на меньшую сумму он не показывается.
